.travis.yml

```diff
@@ -10,9 +10,10 @@ notifications:
 node_js:
   - '4'
   - '6'
+  - '7'
```

Comment:
Sorry I meant `8`. As option `node` on line 14 means `latest stable`, I'll revert the file to original.